### PR TITLE
Update copy for HCB International Support

### PR DESCRIPTION
**Pull request by devenjadhav**
This PR updates the copy changes to highlight the new policy regarding accepting new applications for international projects on bank.

### CODE REVIEW

**Comment by devenjadhav** [comment]
@MelSmith104 @maxwofford

**Review by sampoder — CHANGES_REQUESTED** [review]
This looks a bit visually cluttered: 

Screenshot 2022-05-13 at 7 21 55 PM

File: components/bank/form.js [hunk 1]
```diff
@@ -126,15 +127,17 @@ export default function BankApplyForm() {
...
-                {country}
-              </option>
-            ))}
+            <option value="United States">United States (US)</option>
+            <option value="Canada">Canada (CA)</option>
           </Select>
           <HelperText>
-            We're testing out limited support for international organizations,
-            and want to know in advance if you're operating outside the US.
+            We're currently only able to support projects operating out of the
```
Comment (cjdenio) [inline components/bank/form.js:133]: Personally, I find this confusing; Bank is "only able to support projects operating out of the United States or Canada", but I "might be eligible to run on Bank" if I'm outside North America?

Comment (devenjadhav) [reply components/bank/form.js:133]: we will not accept applications outside of the US by default, but we still want a way for people to be able to reach out to us. I wasn't really sure how it should be worded though. 

@cjdenio do you have any suggestions?

**Comment by devenjadhav** [comment]
> This looks a bit visually cluttered:
> 
> Screenshot 2022-05-13 at 7 21 55 PM

@sampoder I think it might be a better idea to just remove that part. We mention that it works only in North America in the apply form. @MelSmith104 is that okay?

File: components/bank/form.js (hunk 1 above)
Comment (bellesea) [reply components/bank/form.js:133]: What about smth like this?

We mostly support projects operating out of the United States or Canada but if you're outside of those countries, and might be eligible to run on Bank, please shoot us an...

**Comment by hackliv** [comment]
For the form, instead saying "projects operating out of the United States or Canada" maybe edit to "projects operating in the United States and Canada"

Also, on our website we say North America (which includes Mexico) and the form says only U.S. and Canada.

If we remove those parentheses on our website, maybe we add an asterisk so that the information is still present somewhere on the page.

File: components/bank/form.js (hunk 1 above)
Comment (MelSmith104) [reply components/bank/form.js:133]: "We're only able to support projects operating out of the US and CA. If you have any questions, please shoot us an email at bank@hackclub.com."

Keeping the ambiguity open is going to just promote the same confusion that's already happening - i.e. people really trying to make their case as to why their project will work. it's not that we don't want to support them, it's the fact that we're pretty locked out of their country's financial system and it's usually not going to work. active hack clubbers will be active on slack = bank team can reach out them personally / privately to see if they'd be interested in using bank for their project. this means that we can still get the sams, belles, and devs operating on bank but remove all of the confusion that's currently surrounding the application process

**Pull request merged**